lib/fastlane_core/project.rb

```diff
@@ -275,4 +276,18 @@ def parsed_info
       @parsed_info
     end
   end
+
+  # @internal to module
+  def self.xcode_list_timeout
+    (ENV['FASTLANE_XCODE_LIST_TIMEOUT'] || 10).to_i
+  end
+
+  # runs the specified command and kills it if timeouts
+  # @raises Timeout::Error if timeout is passed
+  # @returns the output
+  # FIXME: this is not working exactly as it should. gym#143
+  def self.run_command(command, timeout: 0)
```

Comment:
I described the other method as "internal to the module". This one was meant to as well.

I can move it out of the FastlaneCore scope, but I still want to make it somewhat 'private'

Is there any guideline on how to properly implement stuff that one wants clearly not exposed (not used by users of the spaceship gem) yet still available for unit testing ?